src/style.css

```diff
@@ -0,0 +1,56 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+  font-style: normal;
+  font-weight: 300;
+}
```

Comment:
you need to connect one more font with different font-weight, you have a hint in 7'th point in checklist